osfoffline/client/osf.py

```diff
@@ -59,8 +67,21 @@ def from_data(cls, request_session, data):
     @classmethod
     def load(cls, request_session, *args, **kwargs):
         resp = request_session.get(cls.get_url(*args, **kwargs), params={'page[size]': 250})
+        if (resp.status_code % 500) <= 100:
```

Comment:
Right, but that would be covered by the inequality:

`(502 >= 500 and 500>=500) == True`
`resp.status_code >= 500`

I'm not seeing the functional/readability advantage of the modulo, I guess. (though I'll admit that it's one of my favorite fun operators)